machine_management/managing-infrastructure-machinesets.adoc

```diff
@@ -0,0 +1,66 @@
+[id="managing-infrastructure-machinesets"]
+= Managing infrastructure machine sets
+include::modules/common-attributes.adoc[]
+:context: managing-infrastructure-machinesets
+
+toc::[]
+
+Infrastructure machine sets provide a way to create and manage infrastructure nodes, which allow you to isolate infrastructure workloads. Managing your infrastructure workloads on infra nodes has several benefits.
+
+include::modules/infrastructure-components.adoc[leveloffset=+1]
+
+== Workload placement for infra nodes
+
+The two primary reasons for infra nodes include:
+
+* Preventing incurring billing costs against subscription counts.
+* Separating maintenance workloads and management workloads.
+
+After creating an infrastructure machine set, the new hosted infrastructure nodes have both the `worker` and `infra` roles applied to them. The `worker` role is required when the node is created, and is also used during node upgrades. Nodes with the `infra` role applied are not counted toward the total number of subscriptions that are required to run the environment, even when the `worker` role is also applied. The `worker` role can be removed, but it requires additional procedures to reconfigure the associated machine config pool to manage node upgrades.
+
+[IMPORTANT]
+====
+Unlike earlier versions of {product-title}, you cannot move the infrastructure components to the control plane machines. To move the components, you must create a new machine set.
+====
+
+With an infra node being assigned as a worker, there is a chance user workloads could get inadvertently assigned to an infra node. To prevent other workloads from being scheduled onto infrastructure nodes, there are two possible workflows:
+
+* Apply a taint to the infrastructure nodes and tolerations to the desired infrastructure workloads.
+* Apply a completely separate label to your other nodes and matching node selector to your other workloads such that they are mutually exclusive from infrastructure nodes.
+
+This choice mostly depends on the infrastructure components you plan to assign to the infra nodes. Several infrastructure components do not support configuring taints and tolerations; in those cases, you must apply a separate label to your other nodes so they are mutually exclusive from infrastructure nodes.
```

Comment:
We should document which 'infra components' don't support this. The list of infra components is not 'long'. Our subscription guides outline which components those are clearly: 

- Red Hat OpenShift-included registry
- Router
- OpenShift cluster monitoring
- OpenShift log aggregation
- Red Hat Quay
- Red Hat OpenShift Container Storage
- Red Hat Advanced Cluster Manager

So which of these do you have to use the separate label situation with? 
Honestly if 'one' of these doesn't support 'taints and tolerations' we should probably be promoting/using only the labeling method.